apps/molgenis-components/src/components/forms/RowEdit.vue

```diff
@@ -113,6 +117,12 @@ export default {
   },
   methods: {
     getPrimaryKey,
+    columnLabel(column) {
+     return getLocalizedLabel(column);
+    },
+    columnDescription(column) {
+     return getLocalizedDescription(column);
+    },
```

Comment:
function names should describe what they do, in this case, getColumnLabel and getColumnDescription